src/main/java/org/broadinstitute/consent/http/service/DataAccessRequestService.java

```diff
@@ -245,6 +245,8 @@ private Document createDocumentFromDar(DataAccessRequest d) {
         Document document = Document.parse(gson.toJson(d.getData()));
         document.put(DarConstants.DATA_ACCESS_REQUEST_ID, d.getId());
         document.put(DarConstants.REFERENCE_ID, d.getReferenceId());
+        document.put(DarConstants.CREATE_DATE, d.getCreateDate());
+        document.put(DarConstants.SORT_DATE, d.getCreateDate());
```

Comment:
Still need this one change:
```suggestion
        document.put(DarConstants.SORT_DATE, d.getSortDate());
```